### PR TITLE
Modify codes

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
## Description

Update the pending/dropped codes for v700

### Implementation

- [x] **ready for review**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -2247,27 +2247,28 @@ Hence, we recommend calling `GetPendingTxn` around 1-2 transaction epochs after
...
 
-**Unknown Transactions**
+**Dispatched Transactions**
 
-| `code` | Transaction Status     |
-| ------ | ---------------------- |
-| 4      | Transaction not found  |
+| `code` | Transaction Status                      |
+| ------ | --------------------------------------- |
+| 1      | Transaction was recieved by the lookup  |
```
Comment (bb111189) [inline source/index.html.md:2271]:
```suggestion
| 1      | Transaction was received by the lookup  |
```

Comment (bb111189) [reply source/index.html.md:2271]: @kaikawaliu  need to ensure we don't have similar typo on mongo side

**Review by bb111189 — CHANGES_REQUESTED** [review]
check commet

File: source/index.html.md (hunk 1 above)
Comment (KaustubhShamshery) [reply source/index.html.md]: Did u mean JS side ?

**Review by bb111189 — APPROVED** [review]

File: source/index.html.md (hunk 1 above)
Comment (bb111189) [reply source/index.html.md]: yes

**Pull request merged**